### PR TITLE
switched both live examples and fixed the order, Fixes #2383

### LINKED ISSUE #2383: Issue with "Ordering Flex Items": (short summary here please)
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items

#### What information was incorrect, unhelpful, or incomplete?
i think the first coding example should be where the second coding example is, basically the two are swapped around
(sory nwe user, last one automatically closed, or i did something)
#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/css_flexible_box_layout/ordering_flex_items`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/css/css_flexible_box_layout/ordering_flex_items/index.html
* Last commit: https://github.com/mdn/content/commit/6632228acdde81a315ebefcdad7da232abe28ddf
* Document last modified: 2021-02-03T16:06:28.000Z

</details>

### PR DESCRIPTION
**Pull request by CreaTorAlexander**
Switched both live examples. Also fixed the order was twice 1 and 4.
Fixes #2383

### CODE REVIEW

File: files/en-us/web/css/css_flexible_box_layout/ordering_flex_items/index.html [hunk 1]
```diff
@@ -77,19 +77,19 @@ <h2 id="The_order_property">The order property</h2>
 <ul>
```
Comment (rachelandrew) [inline files/en-us/web/css/css_flexible_box_layout/ordering_flex_items/index.html:77]: I don't understand what you have done here, the existing values in the list are correct in terms of the example. They match the live example as far as I can see.

Comment (CreaTorAlexander) [reply files/en-us/web/css/css_flexible_box_layout/ordering_flex_items/index.html:77]: I don't understand why is it 1 1 2 3 3?
I though it should be consecutively 0 1 2 3 4

Comment (CreaTorAlexander) [reply files/en-us/web/css/css_flexible_box_layout/ordering_flex_items/index.html:77]: NVM, I know understand it is grouping the elements.

**Review by rachelandrew — APPROVED** [review]
thanks!

**Pull request merged**